pcapfile/test/savefile_test.py

```diff
@@ -106,6 +106,8 @@ def test_basic_import(self):
         self.assertTrue(self.capfile.valid, 'invalid capture file')
         self.assertEqual(len(self.capfile.packets), 23,
                          'wrong number of packets!')
+        self.assertEqual(self.capfile.__length__(), 23,
+                         '__length__ not reporting correct number of packets')
```

Comment:
One minor thing, it'd probably be good to also test `len(self.capfile)` to make sure it's behaving correctly.